pwa/src/templates/templateParts/cronjobsForm/CreateCronjobsFormTemplate.tsx

```diff
@@ -0,0 +1,102 @@
+import { faFloppyDisk } from "@fortawesome/free-solid-svg-icons";
+import { useQueryClient } from "react-query";
+import { useCronjob } from "../../../hooks/cronjob";
+
+interface CreateCronjobFormTemplateProps {
+  cronjobId?: string;
+}
+
+export const CreateCronjobFormTemplate: React.FC<CreateCronjobFormTemplateProps> = ({ cronjobId }) => {
+  const { t } = useTranslation();
+  const API: APIService | null = React.useContext(APIContext);
+  const [loading, setLoading] = React.useState<boolean>(true);
+  const [formError, setFormError] = React.useState<string>("");
+
+  const queryClient = useQueryClient();
+  const _useCronjobs = useCronjob(queryClient);
+  const createOrEditCronjob = _useCronjobs.createOrEdit(cronjobId);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit = (data: any): void => {
+    createOrEditCronjob.mutate({ payload: data, id: cronjobId });
+  };
+
+  return (
+    <div className={styles.container}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <section className={styles.section}>
+          <Heading1>{t("Create Cronjob")}</Heading1>
+
+          <div className={styles.buttons}>
+            <Button className={styles.buttonIcon} type="submit" disabled={loading}>
+              <FontAwesomeIcon icon={faFloppyDisk} />
+              {t("Save")}
+            </Button>
+          </div>
+        </section>
+        {formError && <Alert text={formError} title={t("Oops, something went wrong")} variant="error" />}
+        <div className={styles.gridContainer}>
+          <div className={styles.grid}>
+            <FormField>
+              <FormFieldInput>
+                <FormFieldLabel>{t("Name")}</FormFieldLabel>
+                <InputText {...{ register, errors }} name="name" validation={{ required: true }} disabled={loading} />
+              </FormFieldInput>
+            </FormField>
+
+            <FormField>
+              <FormFieldInput>
+                <FormFieldLabel>{t("Description")}</FormFieldLabel>
+                <Textarea
+                  {...{ register, errors }}
+                  name="description"
+                  validation={{ required: true }}
+                  disabled={loading}
+                />
+              </FormFieldInput>
+            </FormField>
+
+            <FormField>
+              <FormFieldInput>
+                <FormFieldLabel>{t("Crontab")}</FormFieldLabel>
+                <InputText
+                  {...{ register, errors }}
+                  name="crontab"
+                  validation={{ required: true }}
+                  disabled={loading}
+                />
+              </FormFieldInput>
+            </FormField>
+
+            <FormField>
+              <FormFieldInput>
+                <FormFieldLabel>{t("Throws")}</FormFieldLabel>
+                <InputText
+                  {...{ register, errors }}
+                  name={"throws"}
```

Comment:
Whenever setting a `string` as a prop, we don't need to wrap it in `{}`.

`name="throws"` works.